app/services/idv/actions/verify_document_status_action.rb

```diff
@@ -1,126 +0,0 @@
-module Idv
-  module Actions
-    class VerifyDocumentStatusAction < Idv::Steps::DocAuthBaseStep
-      def call
-        process_async_state(async_state)
-      end
-
-      def self.analytics_submitted_event
-        :idv_doc_auth_verify_document_status_submitted
```

Comment:
I don't see `AnalyticsEvents#idv_doc_auth_verify_document_status_submitted` called anywhere else. Should it be? And if not, should the method be removed?

Reply:
Removed this, thanks.